prestoadmin/server.py

```diff
@@ -353,18 +353,33 @@ def check_server_status(client):
     Returns:
         True or False
     """
-    result = True
     time = 0
+
+    if len(get_coordinator_role()) < 1:
+        warn('No coordinator defined.  Cannot verify server status.')
+    client = PrestoClient(get_coordinator_role()[0], env.user)
+    node_id = lookup_string_config('node.id', os.path.join(constants.REMOTE_CONF_DIR, 'node.properties'), env.host)
     while time < RETRY_TIMEOUT:
-        result = client.execute_query(SYSTEM_RUNTIME_NODES)
-        if not result:
+        try:
```

Comment:
i can do that in a separate pr.  I don't want to delay this getting in since without this fix we can't run any tests.